instrumentation/scala-concurrent/scala-concurrent.gradle

```diff
@@ -0,0 +1,40 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+apply from: "$rootDir/gradle/test-with-scala.gradle"
+apply plugin: 'org.unbroken-dome.test-sets'
+
+muzzle {
+  pass {
+    group = 'org.scala-lang'
+    module = "scala-library"
+    versions = "[2.8.0,2.12.0)"
+    assertInverse = true
+  }
```

Comment:
cc: @tylerbenson, it looks like muzzle in dd-trace-java is only ever checking `2.11.12` b/c of

```
resolutionStrategy {
  force deps.scala
}
```

in `test-with-scala.gradle`

Reply:
did you find a way to solve it?

Reply:
I guess that's not a problem here since you aren't forcing that version.  Thanks for calling it out.

Reply:
I took the liberty: https://github.com/DataDog/dd-trace-java/pull/1810

Reply:
It's actually failing for me on `latestDepTestCompile` which you seem to have removed here.

Reply:
oh, good point, i added it back